python/test/openmldb_client_test.py

```diff
@@ -176,7 +181,28 @@ def check_result(self, rs, expectRows, orderIdx = 0):
     for row in rs:
       self.assertEqual(row, expectRows[i], "not equal row: {}\n{}".format(row, expectRows[i]))
       i+=1
-    
+
+  def check_exectute_many(self,connection,sql):
+    try:
+      connection.execute(sql,[{"col1":1005, "col2":"2020-12-29", "col3":"shandong", "col4":"jinan", "col5":6},
+                              {"col1":1006, "col2":"2020-12-30", "col3":"fujian", "col4":"fuzhou", "col5":7}]);
+    except Exception as e:
+      self.assertTrue(False)
```

Comment:
It seems that it always throws exception and trigger the code `self.assertTrue(False)` which will fail the tests.

Maybe we should change the code like this.

```
    try:
      connection.execute(sql,[{"col1":1005, "col2":"2020-12-29", "col3":"shandong", "col4":"jinan", "col5":6},
                              {"col1":1006, "col2":"2020-12-30", "col3":"fujian", "col4":"fuzhou", "col5":7}]);
      self.assertTrue(False)
    except Exception as e:
      pass
```

Reply:
Updated.